src/e2e/java/teammates/e2e/cases/e2e/FeedbackConstSumOptionQuestionE2ETest.java

```diff
@@ -100,10 +99,8 @@ private void testSubmitPage() {
         AppUrl url = createUrl(Const.WebPageURIs.SESSION_SUBMISSION_PAGE)
                 .withUserId(student.googleId)
                 .withCourseId(student.course)
-                .withSessionName(feedbackSession.getFeedbackSessionName())
-                .withRegistrationKey(getKeyForStudent(student));
+                .withSessionName(feedbackSession.getFeedbackSessionName());
```

Comment:
I think you'd agree that the time is ripe now to create a base class for all these `Feedback*QuestionE2ETest`.